spec/music_album_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rspec'
+require_relative '../App/MusicAlbum/music_album'
+
+def capture_stdout
+  $stdout = StringIO.new
+  yield
+  $stdout.string
+end
+
+RSpec.describe MusicAlbum do
+  before(:each) do
+    MusicAlbum.class_variable_set(:@@all_albums, [])
+    MusicAlbum.class_variable_set(:@@unique_genres, Set.new)
+  end
+
+  describe '.add_album' do
+    it 'agrega un álbum a la lista de álbumes' do
+      allow(MusicAlbum).to receive(:gets).and_return("Album Title\n", "Artist Name\n", "2023-09-21\n", "y\n", "Genre\n")
+      MusicAlbum.add_album
+      expect(MusicAlbum.class_variable_get(:@@all_albums)).not_to be_empty
+    end
+  end
+
+  describe '.list_albums' do
+    it 'muestra la lista de álbumes' do
+      album = MusicAlbum.new('Test Album', 'Test Artist', '2023-09-21', true, 'Test Genre', true)
+      MusicAlbum.class_variable_get(:@@all_albums) << album
+
+      output = capture_stdout { MusicAlbum.list_albums }
+
+      expect(output).to include('Test Album')
+      expect(output).to include('Test Artist')
+      expect(output).to include('Yes')
+      expect(output).to include('Test Genre')
+      expect(output).to include('Yes')
+    end
+  end

```

Comment:
- [ ] @Team Great work adding tests for the music_album, however, you only added tests for only 2 methods out of 10 methods present, kindly ensure all methods are tested.

Reply:
finished! ✔️

Reply:
> * [x]  @Team Great work adding tests for the music_album, however, you only added tests for only 2 methods out of 10 methods present, kindly ensure all methods are tested.

- [ ] Here also, Just as suggested by the previous reviewer, We need to test methods inside your classes, Here I can see the `can_be_archived` method is not tested among others, Please make sure to test some of the main methods inside your `Music_album` class including the `can_be_archived`.